pumps/elasticsearch.go

```diff
@@ -143,6 +154,10 @@ func (e *ElasticsearchPump) getOperator() (ElasticsearchOperator, error) {
 		httpClient = &http.Client{Transport: &ApiKeyTransport{APIKey: conf.AuthAPIKey, APIKeyID: conf.AuthAPIKeyID}}
 	}
 
+	if conf.UseSSL {
+		httpClient = &http.Client{Transport: &http.Transport{TLSClientConfig: e.esConf.tlsConfig}}
```

Comment:
Instead of: `httpClient = &http.Client{Transport: &http.Transport{TLSClientConfig: e.esConf.tlsConfig}}` lets do something like:

`httpClient = &http.Client{Transport: &http.Transport{TLSClientConfig: e.esConf.getTLSConfig()}}`

Reply:
I'll do something similar because we need to check if GetTLSConfig() returns an error